routes/storeRouter.js

```diff
@@ -71,9 +71,11 @@ router.post('/', jwtVerify, async (req, res) => {
   const { store_name, store_url } = store;
 
   // check if those parameters exist, if not - reject
-  if (!store_name) {
+  if (!store_name || store_name === '') {
```

Comment:
Good on you for checking what the state could potentially send in with no user input incase something goes south. User forget to title store. I would like to see this error message displayed on the front end to alert the user.